src/main/java/net/atos/zac/app/informatieobjecten/converter/RESTInformatieobjectConverter.java

```diff
@@ -380,8 +385,19 @@ public List<RESTEnkelvoudigInformatieobject> convertUUIDsToREST(
             final Zaak zaak
     ) {
         return enkelvoudigInformatieobjectUUIDs.stream()
-                .map(enkelvoudigInformatieobjectUUID -> convertToREST(
-                        drcClientService.readEnkelvoudigInformatieobject(enkelvoudigInformatieobjectUUID), zaak))
+                .map(enkelvoudigInformatieobjectUUID -> {
+                    try {
+                        return convertToREST(
+                                drcClientService.readEnkelvoudigInformatieobject(enkelvoudigInformatieobjectUUID), zaak);
+                    } catch (FoutException e) {
```

Comment:
Can't we rename this `FoutException` to something more meaningful maybe?

Reply:
Good idea. This Exception wraps a `Fout`, which kind of implements [Problem Details](https://datatracker.ietf.org/doc/html/rfc7807#section-3.1)
Maybe `ProblemDetails` and `ProblemDetailsException`?

Reply:
Just discussed. We will leave it for now.